app/views/catalog/_sort_and_per_page.html.erb

```diff
@@ -0,0 +1,10 @@
+<div id="sortAndPerPage" class="sort-pagination d-md-flex justify-content-between" role="navigation" aria-label="<%= t('blacklight.search.per_page.aria_label')%>">
+  <%= render partial: "paginate_compact", object: @response if show_pagination? %>
```

Comment:
if you use "partial", don't you also need to use "locals". I prefer to omit partial.

Reply:
This is copied directly from blacklight.

Reply:
I was wrong. This is actually important.